cfme/common/vm_views.py

```diff
@@ -64,6 +64,40 @@ def InstanceEntity():  # noqa
     })
 
 
+class NonEditableTable(Table):
```

Comment:
I don't think the name 'NonEditable' is accurate, I would expect more method overrides in such a table class.

What do you think about 'ClickableTable' or 'SelectTable', something along those lines?

I think this class may be more appropriate defined in WT_manageiq and imported here, because I think there may be other places in the UI where we'll see or use it.

Reply:
I was thinking that this Table won't be popular, but if @mfalesni don't mind it can be moved to WT_manageiq.